plugins/modules/identity/keycloak/keycloak_realm.py

```diff
@@ -0,0 +1,1145 @@
+                description:
+                    - If I(auth) is set to C(True), password for the SMTP server.
+            port:
+                type: int
+                description:
+                    - Port of the SMTP server.
+            replyTo:
+                type: str
+                description:
+                    - Reply-to email address.
+            replyToDisplayName:
+                type: str
+                description:
+                    - Reply-to email address displayed name.
+            ssl:
+                description:
+                    - Specifies whether to use SSL when connecting to the SMTP server.
+                type: bool
+            starttls:
+                description:
+                    - Specifies whether to use STARTTLS when connecting to the SMTP server.
+                type: bool
+            user:
+                type: str
+                description:
+                    - If I(auth) is set to C(True), username for the SMTP server.
+
+    ssl_required:
+        type: str
+        description:
+            - Specifies whether HTTPS is enforced and for what addresses.
+            - C(none) doesn't enforce HTTPS.
+            - C(all) enforces HTTPS on all requests.
+            - C(external) enforces HTTPS on all requests that are not localhost or private IP space.
+        choices:
+            - none
+            - all
+            - external
+        aliases:
+            - sslRequired
+
+    sso_session_idle_timeout:
+        type: int
+        description:
+            - Time an SSO session is allowed to idle before it expires, in seconds.
+        aliases:
+            - ssoSessionIdleTimeout
+
+    sso_session_max_lifespan:
+        type: int
+        description:
+            - Maximum lifetime for an SSO session, in seconds.
+        aliases:
+            - ssoSessionMaxLifespan
+
+    supported_locales:
+        type: list
+        elements: str
+        description:
+            - List of supported locales for this realm. Keycloak (3.4) ships with C(ca), C(de),
+              C(en), C(es), C(fr), C(it), C(ja), C(lt), C(nl), C(no), c(pt-BR), c(ru), c(sv), and
+              c(zh-CN).
+        aliases:
+            - supportedLocales
+
+    verify_email:
+        description:
+            - Specifies whether a user needs to verify their email the first time they log in.
+        type: bool
+        aliases:
+            - verifyEmail
+
+    wait_increment_seconds:
+        type: int
+        description:
+            - For brute force detection, wait increment in seconds per failure.
+        aliases:
+            - waitIncrementSeconds
+
+notes:
+    - Supports I(check_mode).
+    - In the API, a realm representation has further fields which are not covered in this module
+      since they have their own API endpoints and (hopefully) their own Ansible modules. Sorted by
+      ansible_module, they are
+    - M(keycloak_scope) for I(clientScopeMappings) and I(scopeMappings)
```

Comment:
```suggestion
    - M(community.general.keycloak_scope) for I(clientScopeMappings) and I(scopeMappings)
```
All module names in `M(...)` *must* use FQCN. (There are some more below.)